nc-news-frontend/src/pages/Article.jsx

```diff
@@ -21,6 +23,17 @@ const Article = () => {
       fetchItem();
     }, [article_id]);
 
+    useEffect(() => {
+        const fetchItem = () => {
+          getComment(article_id).then((comments) => {
+            setComments(comments);
+           
+            setLoading(false);
+          });
+        };
+        fetchItem();
+      }, [article_id]);
+
```

Comment:
You have two useEffects here which you want to re-run under the same condition. These can and should be combined. 

You are also setting setLoading to be false when either getOneArticle finishes or when getComment loads, but this will not wait for both to finish. Can you think of a way to only setLoading to be false when both of these two fetch requests have happened?